src/client/api/quoteBundleHasAccident.ts

```diff
@@ -0,0 +1,5 @@
+import { QuoteBundle } from 'data/graphql'
+import * as quoteSelector from './quoteSelector'
```

Comment:
Just a question :) Why not directly import `isAccident` here?

Reply:
It's just another way to group the selectors so we don't have to name them all `quoteIsAccidentSelector`